### DISCUSSION TITLE
You need to pay to download Waydroid

### DISCUSSION THREAD

**Discussion by sheerun** [waydroid/waydroid#1297] [General]
Are you aware that download of Waydroid-Linux-jammy-22.04-lineage-18.1-amd64_2023-01-23_1528.iso weights 5.3GB while free quota for mega.co is 5GB? It means one needs to pay to download this file by subscribing to mega.co

**Comment by YAFU** [comment]
Hi.
From where did you find that Mega link?
Quick google search with the name of that file and it is hosted on Sourceforge:
https://sourceforge.net/projects/blissos-dev/files/waydroid/linux/beta/ubuntu/jammy/01-23-2023-gnome/

https://sourceforge.net/projects/blissos-dev/files/waydroid/linux/beta/ubuntu/jammy/01-30-2023-plasma/

EDIT:
Ok I see. The Mega link is the mirror alternative that they offer

**Reply by sheerun** [reply]
From official website: https://waydro.id/#wdlinux (download link, Ubuntu 22.04 Gnome)

**Reply by sheerun** [reply]
@YAFU Thanks for tip with alternative download, didn't notice. Still I think mega should be removed

**Comment by electrikjesus** [comment]
I added the Mega link for users that were complaining about Sourceforge
download speeds. So you just have to pick your poison I guess

**Reply by Lokawn** [reply]
Or pls split the file before upload and it can be downloaded in two days, then can be combined.

**Reply by honguyenminh** [reply]
Can we do a torrent release too? Seems like we have a legitimate, non-sailer, use for it here

**Comment by unrealmy** [comment]
I truly "hate MEGA."
I hate it because there are better sites for downloading files, and while they won't give you the best speeds, at least they don't cut you off with their FUCKING "Free Quota" system, which is a complete lie.
I've had to download classmates' projects for cross-assessment when I was studying. That means I grade theirs, they grade mine, and I also have to download software for both Windows and Linux. I've ALWAYS run into the famous quota, even though I hadn't downloaded anything in months. Needless to say, I'd already reinstalled either the Windows or Linux distribution for a different one, and on two occasions, they'd even changed my modem due to thunderstorms or network failures. So there was NO way my IP was the same as in previous months when I might have downloaded something, enough to eat up my damn quota...
I prefer MediaFire a thousand times over. At least they're more honest about this issue.
Besides, we're trying to give money to the man who got tired of ripping everyone off when his network was called "MegaUpload." Ahh, how short our memory is...

**Reply by unrealmy** [reply]
Sorry for my verbal outbursts...
I sincerely apologize. I got carried away by emotion.

**Reply by electrikjesus** [reply]
When I initially uploaded it, the mega limit was not only 5gb.
